targets/TARGET_ARM_SSG/TARGET_MUSCA_B1/partition/region_defs.h

```diff
@@ -21,6 +21,10 @@
 
 #include "flash_layout.h"
 
+#ifndef BL2
+#define BL2
```

Comment:
Have you captured somewhere that this is a workaround for Mbed CLI 1 that we will nned to remove when migrating to Mbed CLI 2?

Reply:
Issue logged